generated/library_versions.json

```diff
@@ -0,0 +1,3 @@
+{
+    "org.scala-lang:scala3-library_3": "3.4.1"
```

Comment:
I don't think you should use a non-LTS version of Scala unless you can ensure that all versions of Scala (Scala3) can be released once.

Scala has a very unique compilation and loading mechanism, to the point where each revision in Scala2 is not compatible with each other.

The [Long-term compatibility plans for Scala 3](https://www.scala-lang.org/blog/2022/08/17/long-term-compatibility-plans.html) figured out that:

> The Scala 3 version that you are using to compile your code should be treated the same way as any library dependency in your project. You shouldn’t be afraid of updating it. Bumping the patch version (e.g. 3.1.2 -> 3.1.3) is a no-brainer. Bumping the minor version (e.g. 3.1.3 -> 3.2.0) requires the same considerations as changing the minor version of any other dependency, yet is generally advisable.
You should not cross-compile between Scala 3 versions.

So I think that choose LTS version of Scala3 will be better.

What about Scala2? Let it go, it's time to vanish.